protocol/blockchain/transaction/transaction-signature.md

```diff
@@ -0,0 +1,65 @@
+# Transaction Signature
+
+Generally, every input of a [transaction](/protocol/blockchain/transaction) must contain one or more signatures so that the transaction is valid. This applies to any input whose previous output [locking script](/protocol/blockchain/transaction/locking-script) includes one of the following [operation codes](/protocol/blockchain/script#operation-codes-opcodes): `OP_CHECKSIG`, `OP_CHECKSIGVERIFY`, `OP_CHECKMULTISIG`, `OP_CHECKMULTISIGVERIFY`.
```

Comment:
"must contain" is not really true, but in general they do. How about "require"


"Generally, every input of a [transaction](/protocol/blockchain/transaction) must contain one or more signatures so that the transaction is valid."

I feel that this is a bit ambiguous, they must not, and the transaction can be valid without. How about specifying why a signature is used?  How about something like:

"Generally, every input of a [transaction](/protocol/blockchain/transaction) require one or more signature. The signatures enforce (sign) what the transaction looks like and makes it impossible to temper with."